Misc/NEWS.d/next/Library/2023-10-19-11-40-38.gh-issue-111085.hBPpQr.rst

```diff
@@ -0,0 +1 @@
+Fix :class:`asyncio.Timeout` raising an :exc:`AttributeError` over a :exc:`RuntimeError` due to logic bug. Patch by James Hilton-Balfe
```

Comment:
Just FYI: you can also add yourself to `Misc/ACKS` (please, follow the alphabetical order). I couldn't find you there ;)